docs/sphinx/user_guide/feature/view.rst

```diff
@@ -330,6 +330,60 @@ indices by :math:`N`, ::
     }
   }
 
+Index Layout
+^^^^^^^^^^^^
+
+``RAJA::IndexLayout`` is a layout that uses an indexing strategy for each

```

Comment:
```suggestion
``RAJA::IndexLayout`` is a layout that potentially uses a different indexing strategy for each

```
In reality, all RAJA Layouts use an "index strategy" to index into an array of data. The distinguishing characteristic of `IndexLayout` is the fact that the indexing can be different for each dimension, correct?

Reply:
I think the distinguishing feature should actually be that it can use an index list (or lists) that map input indices to some entry in a view.  I don't really think that the fact that you can assign different indexing 'strategies' for each dimension is the central point here.  I tried rewording this paragraph to convey that.

Reply:
I agree that is the main thing. Having to set one for each dimension is not something you had to do before though so it seems noteworthy as well.

Reply:
I re-worded the first paragraph to convey both of the above points.  Let me know if you think more clarification or elaboration on those main points is needed.